submissions/VikTorNik/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="nav">
+                <ul class="nav__menu">
+                    <li class="li-menu"><a class="item-menu" href="#">File</a></li>
+                    <li class="li-menu"><a class="item-menu" href="#">Edit</a></li>
+                    <li class="li-menu"><a class="item-menu" href="#">View</a></li>
+                    <li class="li-menu"><a class="item-menu" href="#">Run </a></li>
+                    <li class="li-menu"><a class="item-menu" href="#">Info</a></li>
+                    <li class="li-menu"><a class="item-menu" href="#">Help</a></li>
+                    <li class="li-menu">
+                        <input type="checkbox" id="checkbox-base" class="checkbox-base">
+                        <label for="checkbox-base" class="label__base">
+                            <img class="img-menu" src="img/popup-button.png" alt="Additional popup menu">
+                        </label>
+                        <div class="content">
+                            <ul>
+                                <div class="content__base">
```

Comment:
[Check your markup](https://validator.w3.org/nu/?doc=https%3A%2F%2Fviktornik.github.io%2FPopUp%2F)

Reply:
Fixed it.